### DISCUSSION TITLE
How to check for priority going, stuck tasks and reminders

### DISCUSSION THREAD

**Discussion by Divwork92** [madrecha/jamku#1281] [QA] [answered]
How to check priority tasks assigned

**Comment by Kishub** [comment, marked as answer]
If task is marked priority wise either Critical or Urgent the background colour of the task will change,  in table format you can view and filter the task based on priority. Also you can filter based on priority by typing "P1"(highest priority) to "P5"(lowest priority) in search box.

![image](https://github.com/user-attachments/assets/d0b65988-c84a-46b4-b670-ed7dc28d2b08)
